### PR TITLE
Update docs to reflect change to priority field default value

### PR DESCRIPTION
**Pull request by impactmass**
Updating the tutorial after changes made in [#2023](https://github.com/reactioncommerce/reaction/pull/2023).  (which is the changing of layout priority default value and using that to allow overriding of templates).

What to review:
* The main comment is in the `plugin-layouts-3.md` file. Check if the description is sufficient.
* Is there any other part of the docs (related to layouts) that needs a similar comment to the one I've added? (so far I've found just that one file)

### CODE REVIEW

**Comment by aaronjudd** [comment]
Is this still [WIP]?  Seems like the updates are relevant?

**Comment by impactmass** [comment]
Thanks Aaron. I'll revisit this after my current ticket.

File: developer/tutorial/plugin-layouts-3.md [hunk 1]
```diff
@@ -69,12 +69,13 @@ We also need to add our layout to the registry via our `register.js`. We are goi
 ```js
 layout: [{
   layout: "coreLayoutBeesknees",
-  workflow: "coreProductWorkflow",
+  workflow: "coreWorkflow",
```
Comment (zenweasel) [inline developer/tutorial/plugin-layouts-3.md]: Are you sure that's correct? Have you tested this? I am pretty sure it was recently changed to `coreProductWorkflow`

Comment (impactmass) [inline developer/tutorial/plugin-layouts-3.md]: I think to keep things focused, I'll split this into two:
1.  This PR will focus on updating Reaction docs based on the work I did on [custom layouts](https://github.com/reactioncommerce/reaction/pull/2023)

2. Another ticket/issue to focus on fixing the inconsistency I noticed in the Layout tutorial (which I discovered while working on *1 above).
Your comment above falls under this second part. So, I'll recreate and test based on your comment separately for *2

**Comment by impactmass** [comment]
@zenweasel please review this again. 
Please check my updated PR description; alongside my response to your last comment above.

**Pull request merged**